### DISCUSSION TITLE
blog/2024/09/13/%E5%B9%BD%E9%BB%98%E5%8F%8C%E4%BC%91/

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#18] [General]
# blog/2024/09/13/%E5%B9%BD%E9%BB%98%E5%8F%8C%E4%BC%91/

人生有多少个三年呢？60岁之后你真的能把自己交的养老金都领回本吗？

https://996.ninja/blog/2024/09/13/%E5%B9%BD%E9%BB%98%E5%8F%8C%E4%BC%91/

**Comment by healthy2bro** [comment]
大哥，您创业成功了记得带带小弟🙇🏼‍♂️🙇🏼🙇🏼‍♀️🙇🏻‍♀️🙇🏻‍♂️🙇🏻🙇🏽‍♀️🙇🏽‍♂️🙇🏽

**Reply by MingGH** [reply]
好的，到时候给你发500万拿去挥霍
